packages/components/src/scroll-lock/stories/index.tsx

```diff
@@ -66,6 +59,23 @@ function ToggleContainer( props ) {
 	);
 }
 
-export const _default = () => {
-	return <Example />;
+export const Default: ComponentStory< typeof ScrollLock > = () => {
+	const [ isScrollLocked, setScrollLocked ] = useState( false );
+	const toggleLock = () => setScrollLocked( ! isScrollLocked );
+
+	return (
+		<StripedBackground>
+			<div>Start scrolling down...</div>
+			<ToggleContainer>
+				<Button variant="primary" onClick={ toggleLock }>
+					Toggle Scroll Lock
+				</Button>
+				{ isScrollLocked && <ScrollLock /> }
+				<p>
+					Scroll locked:{ ' ' }
+					<strong>{ isScrollLocked ? 'Yes' : 'No' }</strong>
+				</p>
+			</ToggleContainer>
+		</StripedBackground>
```

Comment:
I think the point of this component would be better demonstrated (especially in Docs view) if we wrapped it in some divs that represent the "page body" and the "modal". I'll propose that in a follow-up PR once this is merged.